src/sass/_variables.scss

```diff
@@ -8,4 +8,7 @@ $primary: #d58e32;
 
 $text-color: #2a2a2a;
 
+$btn-active-white: #ffffff;
```

Comment:
you should name variables in a more generic way. this one has a `white` in it, so if someone will change its value to eg. `#ff0022` it won't be white anymore causing confusion.

consider `btn-active-border-color/btn-active-text-color` etc.